tests/trainers/java/time.java

```diff
@@ -0,0 +1,68 @@
+/**
+Test to check the Java Timming. This cover only the adquisition
+of the current date in different ways (because is dependent of
+the system ), but not all the operations can be performed
+- Date
+- LocalTimeDate
+- currentTimeMillis
+- nanoTime
+- instant
+*/
+import java.util.*;
+import java.time.*;
+
+public class Test {
+    void date() throws Exception, InterruptedException {
+        Date d1 = new Date();
+        Thread.sleep(100);
+        Date d2 = new Date();
+        if (d2.getTime() < (d1.getTime() + 100))
+            throw new Exception("java.util.Date");
+    }
+
+    void localTimeDate() throws Exception {
+        LocalDateTime now = LocalDateTime.now();
+        if (now.getYear() < 2022)
```

Comment:
```suggestion
        if (now.getYear() > 2020 && now.getYear() < 2100)
```

I would make this test stricter, especially to watch out for the typical `01-01-1970` date that gets often returned, when something goes wrong in the datetime routines.

Reply:
done